app/backplane/power_module/include/c_power_module.h

```diff
@@ -0,0 +1,62 @@
+#ifndef C_SENSOR_MODULE_H
+#define C_SENSOR_MODULE_H
+
+#include "c_sensing_tenant.h"
+
+// F-Core Includes
+#include <f_core/c_project_configuration.h>
+#include <f_core/messaging/c_message_port.h>
+#include <f_core/os/c_task.h>
+#include <f_core/types.h>
+#include <f_core/net/application/c_udp_broadcast_tenant.h>
+
+
+class CPowerModule : public CProjectConfiguration {
+public:
+
+    // TODO(aaron) Break this apart based on telemetry frequency eventually
+    struct __attribute__((packed)) SensorData {
+      NTypes::NSensor::ShuntData RailBattery;
```

Comment:
oh yea we c++ing fr now. more colons is more better.

Reply:
yes